[service.subtitles.opensubtitles] 5.1.0

juokelis opened the pull request:
### Description
service.subtitles.opensubtitles v5.1.0
updated plugin - ported to python 3.0 to run on Matrix
### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

basrieter commented on service.subtitles.opensubtitles/addon.xml:
```diff
@@ -0,0 +1,78 @@
+	<extension point="xbmc.subtitle.module"
+             library="service.py" />
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en">OpenSubtitles</summary>
```
Please use the correct locale identifier. E.g.: `en_GB` instead of `en`. That goes for all `lang=""` attributes.

basrieter commented on service.subtitles.opensubtitles/addon.xml:
```diff
@@ -0,0 +1,78 @@
+- disable anonymous login, users need to register on opensubtitles.org and login in addon settings.
+		</news>
+		<platform>all</platform>
+		<language/>
```
Remove the empty <language /> tag

basrieter commented:
I also wonder if the original author @opensubtitles is aware of this PR. @enen92 do you have insight in how to handle this?

juokelis commented:
Corrected by @basrieter review. 
@opensuntitles has my changes as pull request in their repo. Repo seems abandoned few years ago, though. 
I do not claim author rights, just updated code to run on python 3 (I'm on Matrix, sharing with community possibility to get opensubtitiles for Matrix).
Checks seems to be failing not because of code, because of pipeline problems?
